sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/query.sql

```diff
@@ -0,0 +1,294 @@
+-- Query for ga_derived.www_site_hits_v2
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+WITH get_session_start_time AS (
+  SELECT
+    PARSE_DATE('%Y%m%d', a.event_date) AS date,
+    a.user_pseudo_id || '-' || CAST(
+      (
+        SELECT
+          `value`
+        FROM
+          UNNEST(event_params)
+        WHERE
+          key = 'ga_session_id'
+        LIMIT
+          1
+      ).int_value AS string
+    ) AS visit_identifier,
+    a.user_pseudo_id AS full_visitor_id,
+    MIN(event_timestamp) AS visit_start_time
+  FROM
+    `moz-fx-data-marketing-prod.analytics_313696158.events_*` a
+  WHERE
+    _TABLE_SUFFIX = FORMAT_DATE('%Y%m%d', @submission_date)
+  GROUP BY
```

Comment:
hint: Use column names for the GROUP BY.

Reply:
Thanks @lucia-vargas-a  - I'm in such a bad habit, I always do the position types, I have to put a post-it reminder on my desk or something LOL